### DISCUSSION TITLE
Request: Provide a container

### DISCUSSION THREAD

**Discussion by andreas-wilm** [GoekeLab/bambu#392] [Q&A] [answered]
Daer authors,

Thanks for making Bambu available! There seem to be a number of installation or package dependency issues currently (e.g. https://github.com/GoekeLab/bambu/issues/380). To avoid this, would it be possible to provider a Docker or Singularity container?

Many thanks,
Andreas

**Comment by jonathangoeke** [comment, marked as answer]
Hi @andreas-wilm
You could try the biocontainer version https://biocontainers.pro/tools/bioconductor-bambu
There is also the nf-core pipeline nanoseq that uses bambu: https://nf-co.re/nanoseq/3.1.0, but I'm not sure right now which version of Bambu is used there
Thanks!
Jonathan

**Comment by andreas-wilm** [comment]
Excellent! Thank you @jonathangoeke. This is definitely worthwhile adding to the README!

**Reply by jonathangoeke** [reply]
that is a good idea. I have converted this issue to a discussion so others can find it in the meantime
